scripts/filter.cpp.in

```diff
@@ -45,15 +45,15 @@ IFilter::UniquePointer @FILTER_NAME@::clone() const
   return std::make_unique<@FILTER_NAME@>();
 }
 
-Result<OutputActions> @FILTER_NAME@::preflightImpl(const DataStructure& data, const Arguments& args) const
+Result<OutputActions> @FILTER_NAME@::preflightImpl(const DataStructure& dataStructure, const Arguments& filterParameterValues, const MessageHandler& messageHandler) const
```

Comment:
"args" doesn't mentally connect the FilterParameter as being description of what is needed to the Argument as being where to actually store that input value to pass to the filter. i'm trying to look at this problem from a developers position outside of BQ. The naming should be clear enough to describe what is going on. the generic term `args` is used to mean a whole raft of different ideas in software engineering. I'm wanting something more obvious so that when a developer of DREAM3D is looking at the codes it is immediately obvious what is needed.

I would also advocate changing the class name of "Arguments" to "FilterArguments" or "ParameterValues" (and also change FilterParameter to ParameterDescription). These things are all loosely related and should come off that way through their naming conventions.